managing-data.html

```diff
@@ -154,11 +305,21 @@
 <div class="line"><br /></div> <!-- end #line -->
 </div> <!-- end #line-block -->
 
-</li>
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
 
-<li><p class="first">To attach a file to an image click on the <span class="bold">+</span> and select from the menu.</p>
+<p class="top_link"><a href="#top"><img src="images/back_to_top.png" alt="Image for link to top of page" style="float: right;"/></a></p>
+
+<h2><a class="anchor" id="attach"></a>Attach files</h2>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
 
-<p>Select <span class="bold">Local document ...</span> to upload a file from your local hard drive.</p>
+<p class="first">Click on the <span class="bold">+</span> button to attach a file to an image or other object.</p>
+
+<p>Select <span class="bold">Local document ...</span> from the menu to upload a file from your local hard drive.</p>
```

Comment:
New Experiment below covers OMERO.editor and therefore doesn't apply for 5.1 onwards - don't know if you want to remove it altogether or just stick a '5.0x and earlier only' note on it.